### DISCUSSION TITLE
Github Actions to generate binaries for Mac, Linux, and Windows

### DISCUSSION THREAD

**Discussion by cspotcode** [nodejs/single-executable#59] [Q&A]
Is there a Github Actions template to generate a binary for Mac, Linux, and Windows?  It would be helpful if, given a project that already bundles down to a single file, we could copy-paste an off-the-shelf GH Actions config to generate downloadable native binaries.

**Comment by RaisinTen** [comment]
I'm not aware of such an existing GHA template. Would you like to build one?

**Comment by cspotcode** [comment]
I can't commit to this, but if I create any one-off configs for projects, I
can link them here.

**Comment by yjl9903** [comment]
I am trying doing this thing, but it only works on actions container ubuntu-latest but not windows-latest (see #65).
